inc/plugin.class.php

```diff
@@ -1717,7 +1734,11 @@ public static function getPlugins() {
     * @return boolean
     */
    public static function isPluginLoaded($name) {
-      return in_array($name, self::getPlugins());
+      // Make a lowercase comparison, as sometime this function is called based on
```

Comment:
As this method is called by `Plugin::isActivated()` and `Plugin::isInstalled()`, some calls are now made using result of `isPluginItemType`, which may return plugin name with a different case as the one used for plugin directory.